config/provider.go

```diff
@@ -25,6 +25,9 @@ import "fmt"
 // ConfigurationChangeCallback is called for updates of configuration data
 type ConfigurationChangeCallback func(key string, provider string, configdata interface{})
 
+// Root marks the root node in a Provider
+const Root = ""
```

Comment:
We should think about adding user config types instead of string Root = "", which can be used everywhere. (e.g. Get(RPCModule), Get(LogConfig), etc..